lessons/pydata/eda-univariate-timeseries/index.ipynb

```diff
@@ -0,0 +1,4745 @@
+{
```

Comment:
<p>Uděláme tedy separátní soubory s řešením? Třeba <code>.py</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>Asi jo, nebo notebook.</p><p> </p><p>TODO: oddělím to a dám se do repozitáře</p>

Reply:
<p>Je tam <code>solutions.ipynb</code></p>